internal/compiler/pathutils.rs

```diff
@@ -0,0 +1,528 @@
+// Copyright © SixtyFPS GmbH <info@slint.dev>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-1.1 OR LicenseRef-Slint-commercial
+
+use std::path::{Path, PathBuf};
+
+// Reimplement some Path handling code: The one in `std` is not available
+// when running in WASM!
```

Comment:
Could have been a doc comment

> The one in `std` is not available when running in WASM!

But this is not true, most of the path handling is available in wasm. Only that it consider that path are URLs and so have a different meaning for absolute path.

It's a bit overkill to re-implement the whole thing IMHO